src/databricks/azext_databricks/_params.py

```diff
@@ -0,0 +1,55 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: disable=line-too-long
+# pylint: disable=too-many-lines
+# pylint: disable=too-many-statements
+
+from azure.cli.core.commands.parameters import (
+    tags_type,
+    get_three_state_flag,
+    get_enum_type,
+    get_location_type
+)
+
+# from azext_databricks.action import (
+#     PeeringAddAuthorizations
+# )
+
+
+def load_arguments(self, _):
+
+    with self.argument_context('databricks workspace create') as c:
+        c.argument('workspace_name', options_list=['--name', '-n'], help='The name of the workspace.')
+        c.argument('tags', tags_type)
+        c.argument('location', arg_type=get_location_type(self.cli_ctx))
+        c.argument('managed_resource_group', help='The managed resource group to create. It can be either a name or a resource id.')
+        c.argument('aml_workspace_id', help='Azure Machine Learning workspace id.')
+        c.argument('custom_virtual_network_id', options_list=['--virtual-network-id'], help='Virtual network id.')
+        c.argument('custom_public_subnet_name', options_list=['--public-subnet-name'], help='Public subnet name.')
+        c.argument('custom_private_subnet_name', options_list=['--private-subnet-name'], help='Private subnet name.')
+        c.argument('enable_no_public_ip', arg_type=get_three_state_flag(), options_list=['--no-public-ip'], help='Set true to disallow public IP. Default is false')
+        c.argument('load_balancer_backend_pool_name', help='The name of load balancer backend pool name.')
+        c.argument('load_balancer_id', help='Load balancer id.')

```

Comment:
same as above, load_balancer_id should support both name and id?

Reply:
similar as virtual-network.